ui/app/styles/core/helpers.scss

```diff
@@ -287,6 +287,12 @@ ul.bullet {
 .has-text-align-center {
   text-align: center;
 }
+
+.has-background-success-light {
+  background-color: $green-100 !important;
+  color: $green-700 !important;
+}
+
```

Comment:
I'm also not a big fan of using `!important`, but this was the only way I can think of to make sure the element was green if it had this class. 

Has anyone seen this tag before? If so, I can totally remove this class!!
<img width="456" alt="Screen Shot 2023-01-18 at 2 04 51 PM" src="https://user-images.githubusercontent.com/30884335/213305517-e3a9b630-7547-464e-8821-d420769dffa3.png">

https://bulma.io/documentation/helpers/color-helpers/

Reply:
On a similar note, do we have a class for this? :) 

<img width="603" alt="Screen Shot 2023-01-18 at 2 16 33 PM" src="https://user-images.githubusercontent.com/30884335/213307328-53dd1fbc-4883-4bde-8de8-e0e0ba27e049.png">

Reply:
We don't set background color very often, so I think that with a specific enough selector it should be fine not using important.  

I haven't seen an instance where an item with the `.tag` class hasn't had a grey background. On that note, this feels kind of like a tag to me so maybe we could add this class there. 
something like: 
```js
// core/tags.scss

tag.is-success {
  background-color: $green-100;
  color: $green-700;
}

```

I don't think I've seen anything like the pink URL 🤔

Reply:
I'm not seeing either of those exact colors anywhere in our color definitions. Not sure if a background named class would be appropriate since it's applying color to the text. Might be best to make a couple of one off classes or find the closest green/red(even though that one is pink!) color classes and run it by design.